modules/swagger-core/src/test/scala/BeanValidatorTest.scala

```diff
@@ -29,9 +29,9 @@ class BeanValidatorTest extends FlatSpec with Matchers {
     password.getMaxLength() should be (20)
 
     val minBalance = properties.get("minBalance").asInstanceOf[DoubleProperty]
-    minBalance.getExclusiveMinimum should be(0.1)
+    minBalance.getExclusiveMinimum().booleanValue() should be (true)
```

Comment:
Why did you change the check for the value (0.1) to convert it to a boolean here?